scipy-notebook/Dockerfile

```diff
@@ -74,6 +74,10 @@ RUN conda create --quiet --yes -p $CONDA_DIR/envs/python2 python=2.7 \
 RUN ln -s $CONDA_DIR/envs/python2/bin/pip $CONDA_DIR/bin/pip2 && \
     ln -s $CONDA_DIR/bin/pip $CONDA_DIR/bin/pip3
 
+# Import matplotlib the first time to build the font cache.
+ENV XDG_CACHE_HOME /home/$NB_USER/.cache/
+RUN $CONDA_DIR/envs/python2/bin/python -c "import matplotlib.pyplot"
```

Comment:
I think setting MPL_BACKEND=Agg for this command should do it. I don't think the inline backend will work below, because IPython startup doesn't get loaded for `python -c`.